content/documentation/known-issues/index.adoc

```diff
@@ -28,16 +28,32 @@ No version of Internet Explorer is supported with Kiali. Users may experience so
 
 In order to have the best experience with Kiali you will need to update to use a link:../prerequisites/#_browser_requirements[supported browser].
 
-== Non Deployment-based workloads
```

Comment:
why is this outdated? we support deployments/workloads, which part is incorrect here so it's removed?

Reply:
"In Kiali, we essentially assume, currently, that a Workload comes from a Deployment." this part isn't true anymore, this was a limitation but it isn't anymore. The JIRA mentioned (https://issues.jboss.org/browse/KIALI-1284) is closed since last october.